provisioner/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -0,0 +1,196 @@
+---
```

Comment:
why did you add this without removing the other one? in `workshops/provisioner/roles/manage_ec2_instances/defaults/main/main.yml`

Reply:
Apologies. Mistake on my side with commit. I'll fix.